src/Tribe/Tickets/Tickets.php

```diff
@@ -245,7 +245,7 @@ function __construct() {
 			// Register all Tribe__Events__Tickets__Tickets api consumers
 			self::$active_modules[$this->className] = $this->pluginName;
 
-			add_filter( 'tribe_events_tickets_modules', 		 array( $this, 'modules' 					 )		  );
+			add_filter( 'tribe_events_tickets_modules', array( $this, 'modules' ) );
 			add_action( 'tribe_events_tickets_metabox_advanced', array( $this, 'do_metabox_advanced_options' ), 10, 2 );
 
 			// Admin AJAX actions for each provider
```

Comment:
The current plan is to only allow ticket submission for authenticated users (there's some payment settings that Community Organizers will need to have in place for creating tickets).